app/src/app/[...not_found]/layout.tsx

```diff
@@ -21,7 +21,7 @@ export default function RootLayout({
   children: React.ReactNode;
 }) {
   return (
-    <html lang="">
+    <html lang="" suppressHydrationWarning>
```

Comment:
I'm not sure if this is the best way to handle it. Ideally we fix the root cause, which is that the data-theme property only renders on the client, not the server (which causes performance issues on the client when rendering as it needs to re-render everything the server sent).